File: medical-portal/src/API/RSBC.DMF.MedicalPortal.API/ViewModels/ChefsBundle.cs
using Rsbc.Dmf.CaseManagement.Service;
using Rsbc.Dmf.IcbcAdapter;

namespace RSBC.DMF.MedicalPortal.API.ViewModels;

public class ChefsBundle
{
    public string caseId { get; set; }
    public PatientCase patientCase { get; set; }
    public Driver driverInfo { get; set; }
    public IEnumerable<MedicalCondition> medicalConditions { get; set; }
    public string dmerType { get; set; }
}

File: medical-portal/src/UI/src/app/shared/api/models/chefs-bundle.ts
/* tslint:disable */
/* eslint-disable */
import { Driver } from '../models/driver';
import { MedicalCondition } from '../models/medical-condition';
import { PatientCase } from '../models/patient-case';
export interface ChefsBundle {
  caseId?: string | null;
  driverInfo?: Driver;
  medicalConditions?: Array<MedicalCondition> | null;
  patientCase?: PatientCase;
}
